core/platform/taskqueue/cloud_taskqueue_services.py

```diff
@@ -61,8 +71,15 @@ def create_http_task(
     parent = CLIENT.queue_path(
         feconf.OPPIA_PROJECT_ID, feconf.GOOGLE_APP_ENGINE_REGION, queue_name)
 
+    # In the type annotation below, task is of type Dict[str, Any] because
+    # its structure can vary a lot.
+    # We can see how the proto message for Task is defined. See the link:
+    # https://github.com/googleapis/python-tasks/blob/2f6ae8318e9a6fc2963d4a7825ee96e41f330043/google/cloud/tasks_v2/types/task.py#L29
+    # Create_task either expects this protobuf message or a dictionary having
+    # a similar structure.
+    # This clearly points to the fact that the task is of type Dict[str, Any].
     # Construct the request body.
```

Comment:
This comment can be removed.

Reply:
Removed these lines:
```
    # Create_task either expects this protobuf message or a dictionary having
    # a similar structure.
    # This clearly points to the fact that the task is of type Dict[str, Any].
```

Reply:
Ah sorry for ambiguity, I meant just `# Construct the request body.`.

Reply:
Looking at it the lines that you removed are quite redundant so lets keep them removed.

Reply:
Removed `# Construct the request body.` also.